### PR TITLE
Add default.mixed to the plugin page

### PR DESCRIPTION
**Pull request by josh146**

### CODE REVIEW

File: jinja/plugins.html [hunk 1]
```diff
@@ -65,6 +65,17 @@ <h4 class="card-header juicy-peach-gradient">lightning.qubit (beta)</h4>
                     </div>
                 </a>
             </div>
+            <div class="card plugin-card">
+                <a href="https://pennylane.readthedocs.io/en/stable/code/api/pennylane.devices.default_mixed.DefaultMixed.html">
+                    <h4 class="card-header juicy-peach-gradient">default.mixed</h4>
+                    <div class="card-body">
+                        <p class="card-text">
+                            A density matrix qubit simulator written in Python.
```
Comment (ixfoduap) [inline jinja/plugins.html:73]:
```suggestion
                            A mixed-state qubit simulator written in Python.

```

File: jinja/plugins.html [hunk 2]
```diff
@@ -65,6 +65,17 @@ <h4 class="card-header juicy-peach-gradient">lightning.qubit (beta)</h4>
                     </div>
                 </a>
             </div>
+            <div class="card plugin-card">
+                <a href="https://pennylane.readthedocs.io/en/stable/code/api/pennylane.devices.default_mixed.DefaultMixed.html">
+                    <h4 class="card-header juicy-peach-gradient">default.mixed</h4>
+                    <div class="card-body">
+                        <p class="card-text">
+                            A density matrix qubit simulator written in Python.
+                            <br><br>A good choice for optimizations involving mixed-state computations, for example noisy simulations.
```
Comment (ixfoduap) [inline jinja/plugins.html:74]:
```suggestion
                            <br><br>A good choice for simulating noisy circuits and quantum channels.

```

**Review by ixfoduap — APPROVED** [review]
@josh146 If you like my suggestions, feel free to merge!

**Comment by josh146** [comment]
Thanks @ixfoduap! Your suggestions made the card read better 👍 

This PR will be merged in once the new device is available (after the release).

**Review by ixfoduap — APPROVED** [review]

**Pull request merged**